software/vvveb.yml

```
@@ -0,0 +1,14 @@
+name: "Vvveb CMS"
+website_url: "https://www.vvveb.com"
+description: "Powerful and easy to use cms to build websites, blogs or ecommerce stores."
+licenses:
+  - AGPL-3.0
+platforms:
+  - PHP
+  - Docker
+tags:
+  - Content Management Systems (CMS)
+  - E-commerce
+  - Blogging Platforms
```

```suggestion
  - Content Management Systems (CMS)
```

Traditionally CMS are only listed in this category (https://awesome-selfhosted.net/tags/content-management-systems-cms.html), so I suggest we do the same for this one.

We might want to revisit this later.